### PR TITLE
Ros transforms

### PR DESCRIPTION
**Pull request by EyedBread**

### CODE REVIEW

File: src/dlio/odom.cc [hunk 1]
```diff
@@ -175,15 +174,25 @@ void dlio::OdomNode::getParams() {
...
-  this->imu_frame = ns + "/" + this->imu_frame;
+  // Concatenate Frame Name Strings (if there is one)
+  if (ns.compare("") != 0 && !ns.empty()) {
+    //Append to frames if we have namespace
+    this->odom_frame = ns + "/" + this->odom_frame;
+    this->baselink_frame = ns + "/" + this->baselink_frame;
+    this->lidar_frame = ns + "/" + this->lidar_frame;
+    this->imu_frame = ns + "/" + this->imu_frame;
+  }
+  std::cout << "odom_frame: " << this->odom_frame << std::endl;
```
Comment (aronasefaw) [inline src/dlio/odom.cc:189]: Why are you printing stuff, if it was for debug remove before push to master

File: launch/dlio.launch [hunk 2]
```diff
@@ -21,8 +21,12 @@
   <arg name="pointcloud_topic" default="/os_cloud_node/points"/>
   <arg name="imu_topic" default="/os_cloud_node/imu"/>
 
+  <node pkg="tf2_ros" type="static_transform_publisher" name="static_tf_base_link_to_os_sensor"
+    args="0.5 0 1   1 0 0 0 base_link  os_sensor " />
```
Comment (aronasefaw) [inline launch/dlio.launch]: Where did you get the translations? Seem a bit arbitrary, did you take them from my slack message with all the coords from the cad?

**Review by aronasefaw — CHANGES_REQUESTED** [review]
See comments, also has it been tested on a rosbag?

File: src/dlio/odom.cc (hunk 1 above)
Comment (EyedBread) [reply src/dlio/odom.cc:189]: This will only be printed once, so not a big deal

File: launch/dlio.launch (hunk 2 above)
Comment (EyedBread) [reply launch/dlio.launch]: Yes, a rough approximation, it still worked. However this transform shouldn't be here but on slam_central, gonna update code

**Review by aronasefaw — APPROVED** [review]

**Pull request merged**